content/sensu-go/6.1/operations/control-access/_index.md

```diff
@@ -5,20 +5,143 @@ product: "Sensu Go"
+**NOTE**: For API-specific authentication, see the [API overview](../../api/#access-control) and [Use API keys to authenticate to Sensu](use-apikeys/).
+{{% /notice %}}
 
+Authorization establishes and manages user permissions: the extent of access users have for different Sensu resources.
+Configure authorization with [role-based access control (RBAC)][4] to exercise fine-grained control over how they interact with Sensu resources.
 
-[1]: auth/
-[2]: create-read-only-user/
+## Authentication
+
+Administrators can use Sensu's [built-in basic authentication provider][14] or configure [external authentication providers][15] to authenticate via Lightweight Directory Access Protocol (LDAP), Active Directory (AD), or OpenID Connect 1.0 protocol (OIDC).
+
+### Use built-in basic authentication
+
+Sensu's built-in basic authentication provider allows you to create and manage user credentials (usernames and passwords) with the [users API][53], either directly or using [sensuctl][2].
+The basic authentication provider does not depend on external services and is not configurable.
+
+Sensu records basic authentication credentials in [etcd][54].
+
+### Use an authentication provider
+
+**COMMERCIAL FEATURE**: Access authentication providers in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][6].
+
+In addition to built-in authentication, Sensu includes commercial support for authentication using external authentication providers via [Lightweight Directory Access Protocol (LDAP)][44], [Active Directory (AD)][37], or [OpenID Connect 1.0 protocol (OIDC)][7].
+
+#### Configure authentication providers
+
+**1. Write an authentication provider configuration definition**
+
+* Standards-compliant LDAP tools like OpenLDAP: [LDAP configuration examples][29] and [specification][30]
+* Microsoft AD, including Azure AD: [AD configuration examples][31] and [specification][32]
+* OIDC tools like Okta and PingFederate: [OIDC configuration examples][9] and [specification][12]
+
+**2. Apply the configuration with sensuctl**
+
+Log in to sensuctl as the [default admin user][3] and apply the configuration to Sensu:
+
+{{< code shell >}}
+sensuctl create --file filename.json
+{{< /code >}}
+
+Use sensuctl to verify that your provider configuration was applied successfully:
+
+{{< code shell >}}
+sensuctl auth list
+
+ Type     Name    
+────── ────────── 
+ ldap   openldap  
+{{< /code >}}
+
+#### Manage authentication providers
+
+View and delete authentication providers with the [authentication providers API][27] or these sensuctl commands.
+
+To view active authentication providers:
+
+{{< code shell >}}
+sensuctl auth list
+{{< /code >}}
+
+To view configuration details for an authentication provider named `openldap`:
+
+{{< code shell >}}
+sensuctl auth info openldap
+{{< /code >}}
+
+To delete an authentication provider named `openldap`:
+
+{{< code shell >}}
+sensuctl auth delete openldap
+{{< /code >}}
+
```

Comment:
Consider adding a high-level example of logging in via sensuctl to complete the circle. This gives is a place to add a note similar to `Users _do not_ need to provide the username_prefix when logging in to Sensu.` which is currently only documented for individual authentication providers.